ext/standard/dir.stub.php

```diff
@@ -7,21 +7,21 @@ class Directory
     /**
      * @param resource|null $dir_handle
      * @return void
-     * @alias closedir
+     * @implementation-alias closedir
      */
     public function close($dir_handle = null) {}
 
     /**
      * @param resource|null $dir_handle
      * @return void
-     * @alias rewinddir
+     * @implementation-alias rewinddir
      */
     public function rewind($dir_handle = null) {}
 
     /**
      * @param resource|null $dir_handle
      * @return string|false
-     * @alias readdir
+     * @implementation-alias readdir
```

Comment:
https://www.php.net/manual/en/directory.read.php

> public Directory::read ([ resource $dir_handle ] ) : string
> Same as readdir(), only dir_handle defaults to $this->handle.

Changing this to `@implementation-alias` seems like an improvement - I didn't know this type of aliasing was used in php (readdir() implementation internally checks if $this is set to distinguish between the method and global function)

I don't see anything wrong elsewhere at a glance